components/calendar/index.js

```diff
@@ -110,9 +130,14 @@ const Calendar = React.createClass({
           { preDays }
         </div>
       );
+      toggleCalendarSection = (
+        <button className="cal-button--preDays" onClick={this.preDaysClickHandler}>
+          Skjul tidligere arrangementer
+        </button>
```

Comment:
Most of `toggleCalendarSection` seems to be repeated. Maybe just find a way to toggle the text instead?